Source/Core/Core/IOS/Network/IP/Top.h

```diff
@@ -62,6 +62,7 @@ class NetIPTop : public Device
   NetIPTop(Kernel& ios, const std::string& device_name);
   virtual ~NetIPTop();
 
+  void DoState(PointerWrap& p) override;
```

Comment:
Since you added this, you must not forget to increment the save state version over here:
https://github.com/dolphin-emu/dolphin/blob/099197b480bcc35fd84f059359ab739a30b0bac0/Source/Core/Core/State.cpp#L76-L77